tests/integration-tests/tests/test_efa/test_efa.py

```diff
@@ -86,10 +89,14 @@ def _test_osu_benchmarks(remote_command_executor, scheduler_commands, test_datad
     logging.info("Running OSU benchmarks")
     remote_command_executor.run_remote_script(str(test_datadir / "init_osu_benchmarks.sh"), hide=True)
 
-    result = scheduler_commands.submit_script(str(test_datadir / "osu_submit.sh"), slots=2 * slots_per_instance)
+    result = scheduler_commands.submit_script(
+        str(test_datadir / "osu_submit.sh"), slots=2 * slots_per_instance, filename="osu_submit.sh.o%j"
+    )
     job_id = scheduler_commands.assert_job_submitted(result.stdout)
     scheduler_commands.wait_job_completed(job_id)
     scheduler_commands.assert_job_succeeded(job_id)
 
     # TODO: perform assertions on benchmarks results
-    remote_command_executor.run_remote_command("cat /shared/osu.out")
+    output = remote_command_executor.run_remote_command("cat osu_submit.sh.o{0}".format(job_id)).stdout
+    latency = re.search("0\s+(\d\d)\.", output).group(1)  # noqa: W605
```

Comment:
Minor: have you tried r”0....” to fix the W605?

Also pls remove the todo above

Reply:
changed 👌🏻